cix.sh

```diff
@@ -1,24 +0,0 @@
-#!/bin/bash
-set -euo pipefail
-echo "Running with SQ=$SQ_VERSION for $CI_BUILD_NUMBER"
-
-# Need install because mvn org.sonarsource.scanner.maven:sonarqube-maven-plugin:<version>:sonar in ITs will take artifact from local repo
-mvn install -B -e -V -Dsource.skip=true -Denforcer.skip=true -Danimal.sniffer.skip=true -Dmaven.test.skip=true
-
-#deploy the version built by travis
-CURRENT_VERSION=`mvn help:evaluate -Dexpression="project.version" | grep -v '^\[\|Download\w\+\:'`
```

Comment:
Hopefully we don't need these strange behaviours anymore !

Reply:
We do, for Jenkins 😅.

Reply:
Where ?

Reply:
No, we don't. Sorry, my mistake 😄. We can use the env vars set by the `cirrus-env` script.